documentation/docs/examples/timeline.mdx

```diff
@@ -0,0 +1,153 @@
+- `keyframes` - List of Keyframe to apply Style.
+- `nodeCollection` - Set of nodes to be rendered for the Keyframe.
+- `nodeAppearance` - Styling of the nodes.
+
+### Play timelines
+We use `play(startDate: Date, endDate: Date, durationInMilliSeconds: number)`
+- `startDate` - Keyframe date to start the Playback of Timeline Keyframes.
+- `endDate` - Keyframe date to stop the Playback of Timeline Keyframes.
+- `durationInMilliSeconds` - Number of milli-seconds for the full plackback of Keyframes.
+
+```js runnable
+// import { NodeCollectionBase, DefaultNodeAppearance, PropertyFilterNodeCollection } from '@cognite/reveal';
+// import { TimelineTool, Keyframe } from '@cognite/reveal/tools';
+
+model.setDefaultNodeAppearance(DefaultNodeAppearance.Ghosted);
+
+const timeline = new TimelineTool(model);
+
+const nodes = new AssetNodeCollection(sdk, model);
+const keyframeAssetNode = timeline.createKeyframe(new Date('2021-10-05'));
+keyframeAssetNode.assignStyledNodeCollection(nodes, { renderGhosted: false, outlineColor: NodeOutlineColor.Cyan });
+
+nodes.executeFilter({ assetId: 4222791488928479 });
+const keyframeAssetID = timeline.createKeyframe(new Date('2021-10-06'));
+keyframeAssetID.assignStyledNodeCollection(nodes, { renderGhosted: false, outlineColor: NodeOutlineColor.Orange });
+
+const allPipes = new PropertyFilterNodeCollection(sdk, model);
+allPipes.executeFilter({'PDMS': {'Type': 'PIPE'}});
+const keyframeAllPipes = timeline.createKeyframe(new Date('2021-10-07'));
+keyframeAllPipes.assignStyledNodeCollection(allPipes, { renderGhosted: false, color: [40, 200, 20] });
+
+const stuMechNodes = new PropertyFilterNodeCollection(sdk, model);
+stuMechNodes.executeFilter({'PDMS': {':capDisc': 'MECH', 'Purpose': 'STU' }});
+const keyframeMechNodes = timeline.createKeyframe(new Date('2021-10-08'));
+keyframeMechNodes.assignStyledNodeCollection(stuMechNodes, { renderGhosted: false, color: [80, 100, 120] });
+
+timeline.play(new Date('2021-10-05'), new Date('2021-10-09'), 25000);
+```
+
+### Stop timeline playback
+
+```js runnable
+timeline.stop();
+```
+
+### Pause timeline
+
+```js runnable
+timeline.pause();
+```
+
+### Resume timeline
+
+```js runnable
+timeline.resume();
+```
+
+### Get Keyframe by Date
+`getKeyframeByDate(date: Date)`
+- `date` - Date of the Keyframe which needs retrived.
+
+```jsx
+const keyframe = timeline.getKeyframeByDate(new Date('2021-10-06'));
+```
+
+### Remove Styles & Node in Keyframes
+`unassignStyledNodeCollection(nodeCollection: NodeCollectionBase, nodeAppearance: NodeAppearance)`
+- `nodeCollection` - Node set to be removed from keyframes.
+- `nodeAppearance` - Styles to be removed from keyframes.
+
+```js runnable
+const keyframe = timeline.getKeyframeByDate(new Date('2021-10-06'));
+
+keyframe.unassignStyledNodeCollection(nodes, { renderGhosted: false, color: [150, 200, 120] })
+```
```

Comment:
Move this to the section before the example (where you document how to assign styles to keyframes).